tests/test_argv.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from pathlib import Path
+from polytracker import taint_dag, ProgramTrace
+
+
+@pytest.fixture
+def set_env_vars(monkeypatch):
+    monkeypatch.setenv("POLYTRACKER_TAINT_ARGV", "1")
+
+
+@pytest.mark.program_trace("test_argv.cpp", input="any")
+def test_argv(set_env_vars, program_trace: ProgramTrace):
+    assert isinstance(program_trace, taint_dag.TDProgramTrace)
+    argv0 = Path("argv[0]")
```

Comment:
Probably would make sense to change the `fd_headers`  all around the codebase to reflect the fact that we're using them for things that don't have a file descriptor. Or create a separate section in the tdag file. Thoughts?

Reply:
Yeah, I'm not sure exactly what is the best way. Maybe just name them taint-sources (and remove the file descriptor). Then, have a separate section (only during runtime?) that keeps the mapping between taint-sources and file descriptors. What do you think?